bank_classes/account_class.rb

```diff
@@ -47,42 +48,65 @@ def self.find(id)
 # ### Error handling
```

Comment:
In the `find` method above, I would recommend using a more meaningful variable name rather than `x`